Classes/TcaDataGenerator/Generator.php

```diff
@@ -91,6 +91,9 @@ public function create()
                 'hidden' => 0,
                 'pid' => $neighborPage,
             ];
+            if ($mainTable === 'tx_styleguide_flex') {
```

Comment:
this is a bit hacky but i don't have a much better solution at hand at the moment. please add at least some comment that this is for a specific tsconfig to rte in flex test.